tests/ltp/patches/ltp_fsync_fsync01.patch

```diff
@@ -0,0 +1,36 @@
++ // Patch Description: Tests were failing with kernel panic in loop filesystem.
++ // So modified the tests to use root file system.
+diff --git a/testcases/kernel/syscalls/fsync/fsync01.c b/testcases/kernel/syscalls/fsync/fsync01.c
+index 9af38dce0..40dd86330 100644
+--- a/testcases/kernel/syscalls/fsync/fsync01.c
++++ b/testcases/kernel/syscalls/fsync/fsync01.c
+@@ -33,6 +33,9 @@ static void verify_fsync(void)
+
+ static void setup(void)
+ {
++       rmdir("mntpoint");
++       SAFE_MKDIR("mntpoint", 0644);
++       SAFE_MOUNT("/dev/vda", "mntpoint", "ext4", 0, NULL);
+        sprintf(fname, "mntpoint/tfile_%d", getpid());
+        fd = SAFE_OPEN(fname, O_RDWR | O_CREAT, 0700);
+ }
+@@ -41,6 +44,10 @@ static void cleanup(void)
+ {
+        if (fd > 0)
+                SAFE_CLOSE(fd);
++       if(tst_umount("mntpoint") < 0){
```

Comment:
space before {, fix the indentation

Reply:
remove the test file mntpoint/tfile_xx before unmounting the drive.